### ISSUE TITLE
Documentation: list of supported controls for each platform

### ISSUE THREAD

**Issue by manfromarce** [unoplatform/uno#5219]
## What would you like clarification on:
Having an overview / table of which UWP controls are supported on other platforms (eg calendarview or mediaelement are not implemented yet, others are implemented everywhere except Skia and so on) would help the community to collaborate where it's most needed and developers to make certain project choices if a control is almost implemented or particularly challenging to port and won't arrive anytime soon.

## Concern?

- [ ] Usage in industry
- [x] Clarification of capabilities
- [x] Getting started with Uno
- [ ] Developing with Uno
- [x] Contributing to the Uno project
- [ ] Publishing your application
- [ ] Support
- [ ] Other (please specify):

## For which Platform:

- [x] iOS
- [x] Android
- [x] WebAssembly
- [x] macOS
- [x] Skia
  - [x] WPF
  - [x] GTK (Linux)
  - [x] Tizen
- [x] Windows

**Comment by davidjohnoliver** [comment]
Is this the sort of thing you're looking for? https://platform.uno/docs/articles/implemented-views.html

**Comment by manfromarce** [comment]
Yes, I didn't notice it but does "All platforms" include Skia (WPF/Linux)? You could also add TabView and controls contained in Windows.Storage.Pickers, like FileOpenPicker and FileSavePicker (I've read somewhere that the last one has been implemented for MacOs).

**Comment by MartinZikmund** [comment]
@manfromarce File pickers are now implemented almost everywhere except GTK, full support matrix here - https://platform.uno/docs/articles/features/windows-storage-pickers.html?q=file%20pickers

**Comment by MartinZikmund** [comment]
All controls are now properly supported, closing
